Common/DTO/BrokerRequests/InternalDeleteUserRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+
+namespace DTO.BrokerRequests
+{
+    public class InternalDeleteUserRequest
+    {
+        public Guid Id { get; set; }
```

Comment:
How about rename this to userid ? I think it is a little mess with user or usercredential id

Reply:
done